spec/helpers/checkouts_helper_spec.rb

```diff
@@ -92,4 +93,13 @@
       expect(content).to include(renew_button)
     end
   end
+
+  describe '#render_renew_all' do
+    let(:content) { helper.render_renew_all }
+
+    it 'renders the right html' do
+      renew_all = '<input type="checkbox" name="renew_all" id="renew_all" value="1" data-select-all="renewal" />'
+      expect(content).to include(renew_all)
```

Comment:
Could get rid of line 98 and just have the expect look at `helper.render_renew_all`.  Kinda curious why you did it this way. 🤔

Reply:
hmm I think I was just keeping it consistent with the other specs so not much a specific reason other than that, I can change it for sure, maybe will change the rest as well then 🤔 what do you think?

Reply:
I guess it makes sense to use it like this when there are more than one example using it, and for this one, yes it does not much

Reply:
so I will get rid of let for the ones that needs it once 👍

Reply:
I wasn't feeling too strong about this one way or another, but, if we did want to _always_ use `let` then I'd have to update some of my specs (for consistency). Thanks.